plugin-server/src/main/ingestion-queues/session-recording/session-recordings-consumer.ts

```
@@ -763,8 +763,9 @@ export class SessionRecordingIngester {
                             }
                         )
                         captureException(err, {
-                            tags: { session_id: sessionManager.sessionId },
+                            tags: { session_id: sessionManager.sessionId, error_context: 'failed-on-flush' },
                         })
+                        throw err
```

here's where we're currently swallowing the error on flush

---

so let's throw instead